tests/sample/bindmonitor_mt_tailcall.c

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// clang -O2 -Werror -c bindmonitor_mt_tailcall.c -o bindmonitor_mt_tailcall_jit.o
+//
+// For bpf code: clang -target bpf -O2 -Werror -c bindmonitor_mt_tailcall.c -o bindmonitor_mt_tailcall.o
+// this passes the checker
+
+// Whenever this sample program changes, bpf2c_tests will fail unless the
```

Comment:
Is there a specific point of failure in bpf2c_tests that fails when this occurs? If so, can we add a specific failure trace to indicate that this is the reason why?

Reply:
Good question.  Did you mean the generated file which contains C driver content, that gets git committed to the source code base?

 I do not know what the challenges behind merging the contents, and the decision made behind it.

@Alan-Jowett  : Can you help share your insight here? Thanks.

Reply:
In the case that another developer updates this program - is there a clear failure which lets the developer know exactly what they need to do here (i.e go run this command and update the generated content)?

Reply:
>(i.e go run this command and update the generated content)?

Yes, that is my understanding too! :)
1. Regenerate
2. Format  (.\scripts\format-code.ps1 -w --files="<file name>"
3. git add/commit